File: Python/1231.py
inStr = input()

for i in inStr:
    if i=="+":
        a, b = map(int, inStr.split("+"))
        print(a + b)
    elif i=="-":
        a, b = map(int, inStr.split("-"))
        print(a - b)
    elif i=="*":
        a, b = map(int, inStr.split("*"))
        print(a * b)
    elif i=="/":
        a, b = map(int, inStr.split("/"))
        if b==0:
            break
        else:
            print("%.2f" % float(a/b))

File: Java/1231.java
import java.util.Scanner;
import java.util.StringTokenizer;

public class Main{
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		
		String inStr = sc.nextLine();
		
		for(int i = 0; i < inStr.length(); i++) {
			
			if(inStr.charAt(i)=='+') {
				StringTokenizer token = new StringTokenizer(inStr, "+");
				int a = Integer.parseInt(token.nextToken());
				int b = Integer.parseInt(token.nextToken());
				System.out.println(a+b);
			}
			
			else if(inStr.charAt(i)=='-') {
				StringTokenizer token = new StringTokenizer(inStr, "-");
				int a = Integer.parseInt(token.nextToken());
				int b = Integer.parseInt(token.nextToken());
				System.out.println(a-b);
			}
			
			else if(inStr.charAt(i)=='*') {
				StringTokenizer token = new StringTokenizer(inStr, "*");
				int a = Integer.parseInt(token.nextToken());
				int b = Integer.parseInt(token.nextToken());
				System.out.println(a*b);
			}
			
			else if(inStr.charAt(i)=='/') {
				StringTokenizer token = new StringTokenizer(inStr, "/");
				int a = Integer.parseInt(token.nextToken());
				int b = Integer.parseInt(token.nextToken());
				System.out.printf("%.2f", (float)a/(float)b);
			}
		}
	}
}